### PR TITLE
Fix Redshift boolean normalization bug

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Our Redshift destination connector was failing normalization when attempting to write any boolean values. The issue is that when converting from the raw table to normalized, we approximately run: `json_extract_path_text('data', "boolean_column_name") as boolean`  which works in every DB supported by DBT except Redshift. In Redshift, converting a text value (the output of the `json_extract_path_text` function) to a boolean directly is not supported; we must first map the text value to a 0/1 int value, then cast that to boolean. This PR does that. 

The opposite of this is true: booleans cannot be converted to text types directly in Redshift. 

This PR introduces fixes for both of these problems.

## How
1. Add DBT macros to convert to/from booleans
2. use them in normalization
3. add test case to std destination to write a boolean value 

## Pre-merge checklist
- [x] bump normalization image version
- [x] update to a new image

## Follow up work: 
1. Create standard destination datatype tests #1662

### CODE REVIEW

File: airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultNormalizationRunner.class);
 
-  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:0.1.3";
+  public static final String NORMALIZATION_IMAGE_NAME = "airbyte/normalization:dev";
```
Comment (sherifnada) [inline airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java:46]: Will change before merging

File: airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java [hunk 2]
```diff
@@ -255,7 +255,7 @@ public void testCheckConnectionInvalidCredentials() throws Exception {
     @Override
     public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
       return Stream.of(
-          Arguments.of("exchange_rate_messages.txt", "exchange_rate_catalog.json"),
+          // Arguments.of("exchange_rate_messages.txt", "exchange_rate_catalog.json"),
```
Comment (ChristopheDuong) [inline airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java:258]: This shouldn't be disabled, right?

**Review by ChristopheDuong — APPROVED** [review]

File: airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java (hunk 2 above)
Comment (sherifnada) [reply airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java]: Correct -- re-enabled it now!

File: airbyte-integrations/bases/base-normalization/Dockerfile [hunk 3]
```diff
@@ -19,5 +19,5 @@ WORKDIR /airbyte
 
 ENTRYPOINT ["/airbyte/entrypoint.sh"]
 
-LABEL io.airbyte.version=0.1.3
+LABEL io.airbyte.version=0.1.5
```
Comment (sherifnada) [inline airbyte-integrations/bases/base-normalization/Dockerfile:22]: 0.1.4 already exists

**Pull request merged**